src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +26,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
```

Comment:
According to the checklist, you should use the classnames library for defining classes conditionally. This helps to make your code cleaner and easier to read. For example, you could use it here to conditionally apply the 'has-text-weight-medium' class.